### PR TITLE
File added : Assignment 6 Part 1 & 2

### PR DESCRIPTION
**Pull request by Akash-ing**

### CODE REVIEW

**Review by vinod2406 — APPROVED** [review]

**Review by akash19972306 — APPROVED** [review]

File: Akash/html/assign 6/Assignment2.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+        <style>
+            div{
+                background-color: teal;
+                text-align: right;
+            }
+        </style>
+    </head>
+    <body>
+        <div><b><i>Professional Details</i></b></div>
+
```
Comment (Pranjali1999) [inline Akash/html/assign 6/Assignment2.html]: Remove extra line

File: Akash/html/assign 6/Assignment1.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            <input type="radio" />
+            <label type="text">Less then 18</label>
+            <input type="radio" />
+            <label type="number">18-25</label>
+            <input type="radio" />
+            <label type="number">26-40</label>
+        </form>
+        <div>
+            <h2>
+                <b><i>Academic Detail</i></b>
```
Comment (Pranjali1999) [inline Akash/html/assign 6/Assignment1.html]: b and i tag is out dated

**Review by Pranjali1999 — CHANGES_REQUESTED** [review]

File: Akash/html/assign 3/Assign9.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+            <li>Aniket</li>
+        </ul>
+
+        <ol>
+            <li>Bipin</li>
+            <li>Priyanka</li>
+            <li>Yashasvi</li>
+        </ol>
+
+
```
Comment (shivling123) [inline Akash/html/assign 3/Assign9.html]: remove extra space

File: Akash/html/assign 3/assign10.html [hunk 4]
```diff
@@ -0,0 +1,21 @@
+<html>
+<head>
+	<title></title>
+    <link rel="stylesheet" href="style.css"></link>
+</head>
+
```
Comment (shivling123) [inline Akash/html/assign 3/assign10.html]: remove extra space

File: Akash/html/assign 3/assign7.html [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+            ol.c{
+                list-style-position: inside;
+            }
+            div{
+                color: red(240, 6, 6, 0.192);
+            }
+            div1{
+                color: tomato;
+            }
+
```
Comment (shivling123) [inline Akash/html/assign 3/assign7.html]: remove extra space

File: Akash/html/assign 3/assign7.html [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+                </ul>
+
+            <li class="b">MI-MEAN-7</li>
+         <ul class="a">
+             <li>
+                 sagar
+             </li>
+             <li>umesh</li>
+
+
```
Comment (shivling123) [inline Akash/html/assign 3/assign7.html]: remove extra space

File: Akash/html/assign 3/assign7.html [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+            <li class="b">MI-MEAN-9</li>
+
+              <ul class="a">
+                <li>Anil</li>
+                <li>Monaj</li>
+                   
+            </ul>
+            
+            
+
```
Comment (shivling123) [inline Akash/html/assign 3/assign7.html]: remove extra space

File: Akash/html/assign 3/assign7.html [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+                        Triveni
+                    </li>
+                    <li>
+                        Kalyani
+                    </li>
+                </ul>
+            </ol>
+            
+            
+
```
Comment (shivling123) [inline Akash/html/assign 3/assign7.html]: remove extra space

File: Akash/html/assign 3/assign8.html [hunk 9]
```diff
@@ -0,0 +1,35 @@
+<html>
+    <head>
+        <title>assignment3</title>
+        <link rel="stylesheet" href="style.css"></link>
+    </head>
+
```
Comment (shivling123) [inline Akash/html/assign 3/assign8.html]: remove extra space

File: Akash/html/assign 3/assign7.html [hunk 10]
```diff
@@ -0,0 +1,97 @@
...
+                    
+         </ul>
+
+            <li>MI-MEAN-8</li>
+            <ul class="a">
+                <li>Mahika</li>
+                <li>pallavi</li>
+                <li>reva</li>   
+            </ul>
+
```
Comment (shivling123) [inline Akash/html/assign 3/assign7.html]: remove extra space

File: Akash/html/assign 3/assign7.html [hunk 11]
```diff
@@ -0,0 +1,97 @@
...
+                <li>pallavi</li>
+                <li>reva</li>   
+            </ul>
+            
+            <li class="b">MI-MEAN-9</li>
+
+              <ul class="a">
+                <li>Anil</li>
+                <li>Monaj</li>
+
```
Comment (shivling123) [inline Akash/html/assign 3/assign7.html]: remove extra space

File: Akash/html/assign 3/assign7.html [hunk 12]
```diff
@@ -0,0 +1,97 @@
...
+            <ol class="d"><num>1.</num>MI-MEAN
+            <ol class="c">
+                <li>MI-MEAN-6</li>
+            <ul>​
+                    <li>Shweta</li>
+                    <li>Ajay</li>
+                    <li>Trupti</li>
+                      
+                </ul>
+
```
Comment (shivling123) [inline Akash/html/assign 3/assign7.html]: remove extra space

File: Akash/html/assign 3/Assign9.html [hunk 13]
```diff
@@ -0,0 +1,39 @@
...
+            .p1 {
+                font-family: "Times New Roman", Times, serif;
+            }
+        </style>
+    </head>
+    <body style="background-color: pink;">
+        <h1 class="p1">Ambersoft</h1><br>
+
+        <p>
+
```
Comment (shivling123) [inline Akash/html/assign 3/Assign9.html]: remove extra space

**Review by shivling123 — CHANGES_REQUESTED** [review]
do changes as soon  as and send push code agai

**Review by vishnukopnar — APPROVED** [review]

**Review by Bhavika222 — APPROVED** [review]

**Review by Pranjali1999 — APPROVED** [review]

File: Akash/html/assign 4/assig2.html [hunk 14]
```diff
@@ -0,0 +1,89 @@
...
+                text-align: center;
+                border-bottom: 2px solid black;
+                border-right: 10px solid grey;
+            }
+
+            td{
+                border: 2px solid black;
+                border-right: 10px solid grey;
+            }
+
```
Comment (shivling123) [inline Akash/html/assign 4/assig2.html]: remove extra space

File: Akash/html/assign 4/assig2.html [hunk 15]
```diff
@@ -0,0 +1,89 @@
...
+            
+        </style>
+    </head>
+
```
Comment (shivling123) [inline Akash/html/assign 4/assig2.html]: remove extra space

File: Akash/html/assign 4/assig2.html [hunk 16]
```diff
@@ -0,0 +1,89 @@
...
+    <body>
+        <h2 style="color: rgb(155, 67, 155);"><center>Jan's Spring Term Afternoon Schedule</center></h2>
+
```
Comment (shivling123) [inline Akash/html/assign 4/assig2.html]: remove extra space

File: Akash/html/assign 4/assig2.html [hunk 17]
```diff
@@ -0,0 +1,89 @@
...
+                <th>4:00</th>
+            </tr>
+
+            <tr>
+                <th>4:30</th>
+                <td> </td>
+            </tr>
+            
+        </table>
+
```
Comment (shivling123) [inline Akash/html/assign 4/assig2.html]: remove extra space

File: Akash/html/assign 4/assign1.html [hunk 18]
```diff
@@ -0,0 +1,43 @@
...
+                 <td>0.003</td>
+                 <td>40%</td>
+             </tr>
+            <tr>
+                <th>females</th>
+                <td>1.7</td>
+                <td>0.002</td>
+                <td>43%</td>
+            </tr>
+
```
Comment (shivling123) [inline Akash/html/assign 4/assign1.html]: remove extra space

File: Akash/html/assign 4/assign1.html [hunk 19]
```diff
@@ -0,0 +1,43 @@
...
+        
+
```
Comment (shivling123) [inline Akash/html/assign 4/assign1.html]: remove extra space

File: Akash/html/assign 4/assign3.html [hunk 20]
```diff
@@ -0,0 +1,77 @@
...
+        </style>
+    </head>
+
+    <body>
+        <table>
+
+            <tr>
+                <th rowspan="3">Day</th>
+                <th colspan="3">Seminar</th>
+            </tr>
```
Comment (shivling123) [inline Akash/html/assign 4/assign3.html]: remove extra space
all places
and format properly

File: Akash/html/assign 4/assign3.html [hunk 21]
```diff
@@ -0,0 +1,77 @@
...
+
+
+            <tr>
+                <td>Wednesday</td>
+                <td style="background-color: khaki;">8:00 a.m.</td>
+                <td style="background-color: rgb(120, 235, 243);">12:00 p.m.</td>
+                <td>XSL Formatting Objects</td>
+            </tr>
+
```
Comment (shivling123) [inline Akash/html/assign 4/assign3.html]: remove extra space

**Review by shivling123 — CHANGES_REQUESTED** [review]
do changes as per comment 
push filr again asap

File: Akash/html/assign 2/assign2.html [hunk 22]
```diff
@@ -2,50 +2,71 @@
...
-       <br>
-    They called this search engine backrub.
-     <hr>
-     <h2>Intial public offering</h2>
-     <P>Googel's intial offring (IPO)tooks place five years later,on 19 Augest 2004.<br>
-    At thats larry page,sergy engine birn,and Eric schmidt agreed to work toghster at Google for <strike>30</strike><ims>20</ims><br>
-years, unitl the year 2024.</P>
-<pre>Thank You!!</pre>
-<div>
-
```
Comment (shivling123) [inline Akash/html/assign 2/assign2.html]: Kishanjbethur96

File: Akash/html/assign 2/assign2.html [hunk 23]
```diff
@@ -2,50 +2,71 @@
...
-    <h1 style="text-align: center;">Google</h1>
-    <br>
-    <p><i>This articel is about the company.</i></p> 
-    <h2>From the grage to the Googleplex</h2>
-    <h3>The Google story begins in 1995 at<b>Stanford university.</b><br>
-        <i><b>Larry page</b></i>was considearing stanford for the gard school and<i><b>Sargey brain,</b></i>a student there, was assigned to show him around.</h3>
-        <p>By some accounts,they dissagred about nearly everything during that frist meeting,but by the following year thry struck a <mark>patnership</mark>.<br>
-            Working about their drom rooms, they built a search engine thats used links to determine the importance of individual pages on the work.<a href="https://www.World Wide web/">World Wide Web</a>
-    <br>
```
Comment (shivling123) [inline Akash/html/assign 2/assign2.html]: format all code properly

File: Akash/html/assign 3/Assign9.html [hunk 24]
```diff
@@ -32,8 +32,5 @@ <h1 class="p1">Ambersoft</h1><br>
             <li>Priyanka</li>
             <li>Yashasvi</li>
         </ol>
-
-
```
Comment (shivling123) [inline Akash/html/assign 3/Assign9.html]: format all code properly

File: Akash/html/assign 3/assign7.html [hunk 25]
```diff
@@ -5,24 +5,21 @@
...
-            div{
+            div {
                 color: red(240, 6, 6, 0.192);
             }
-            div1{
+            div1 {
                 color: tomato;
             }
-
-
```
Comment (shivling123) [inline Akash/html/assign 3/assign7.html]: remove space

File: Akash/html/assign 3/assign7.html [hunk 26]
```diff
@@ -36,16 +33,13 @@ <h1><p class="p1">Ambersoft</p></h1>
...
-            <li class="b">MI-MEAN-7</li>
+                <li class="b">MI-MEAN-7</li>
          <ul class="a">
              <li>
                  sagar
              </li>
              <li>umesh</li>
-
-
```
Comment (shivling123) [inline Akash/html/assign 3/assign7.html]: remove space

File: Akash/html/assign 3/assign7.html [hunk 27]
```diff
@@ -61,15 +55,8 @@ <h1><p class="p1">Ambersoft</p></h1>
                 <li>Monaj</li>
                    
             </ul>
-            
-            
-
```
Comment (shivling123) [inline Akash/html/assign 3/assign7.html]: remove space

File: Akash/html/assign 4/assign1.html [hunk 28]
```diff
@@ -1,43 +1,45 @@
...
-                 <td>40%</td>
-             </tr>
             <tr>
                 <th>females</th>
                 <td>1.7</td>
                 <td>0.002</td>
                 <td>43%</td>
             </tr>
-
-
```
Comment (shivling123) [inline Akash/html/assign 4/assign1.html]: remove space and format code

File: Akash/html/assign 4/assign3.html [hunk 29]
```diff
@@ -46,32 +46,29 @@
...
             <tr>
                 <td>Wednesday</td>
-                <td style="background-color: khaki;">8:00 a.m.</td>
-                <td style="background-color: rgb(120, 235, 243);">12:00 p.m.</td>
+                <td style="background-color: khaki">8:00 a.m.</td>
+                <td style="background-color: rgb(120, 235, 243)">12:00 p.m.</td>
                 <td>XSL Formatting Objects</td>
             </tr>
-
-
```
Comment (shivling123) [inline Akash/html/assign 4/assign3.html]: remove space

File: Akash/html/assign 5/assign1.html [hunk 30]
```diff
@@ -11,31 +11,22 @@
...
-            p{
+            p {
                background-color: orange;
             }
-            ul{
+            ul {
                 background-color: orange;
             }
-            
-
```
Comment (shivling123) [inline Akash/html/assign 5/assign1.html]: remove space

**Review by shivling123 — CHANGES_REQUESTED** [review]
format all files code properly and remove all extra spaces of files

File: Akash/html/assign 6/assignment2.html [hunk 31]
```diff
@@ -0,0 +1,67 @@
...
+                <input type="file" id="myfile" name="myfile" multiple />
+                <br /><br />
+                <label for="comment"><strong>Comments:</strong></label>
+                <textarea></textarea>
+            </fieldset>
+            <br />
+            <button class="btn">submit</button>
+        </form>
+    </body>
+</html>
```
Comment (SnehaMakade) [inline Akash/html/assign 6/assignment2.html:67]: add new end line

File: Akash/html/assign 6/Assignment1.html [hunk 32]
```diff
@@ -0,0 +1,59 @@
...
+    <body>
+        <div>
+            <h1>MI-9 intern HTML Assesment</h1>
+        </div>
+        <br />
+        <hr />
+        <div class="a"><h2>personal Details</h2></div>
+        <form>
+            <label for="name">First name :</label>
+            <input type="text" name="frist name" placeholder="frist name" /><br />
```
Comment (SnehaMakade) [inline Akash/html/assign 6/Assignment1.html:25]: placeholder should be like First name do this change in each input box

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
do changes as per comments given

File: Akash/html/assign 3/Assign9.html [hunk 33]
```diff
@@ -0,0 +1,39 @@
+
+<html>
+    <head>
+        <style>
+            h1{text-align: center;}
```
Comment (shivling123) [inline Akash/html/assign 3/Assign9.html]: keep space between bracket and element

File: Akash/html/assign 3/Assign9.html [hunk 34]
```diff
@@ -0,0 +1,39 @@
...
+            p{text-align: center;}
```
Comment (shivling123) [inline Akash/html/assign 3/Assign9.html]: keep space between bracket and element

File: Akash/html/assign 3/Assign9.html [hunk 35]
```diff
@@ -0,0 +1,39 @@
...
+            .p1 {
+                font-family: "Times New Roman", Times, serif;
+            }
+        </style>
+    </head>
+    <body style="background-color: pink;">
+        <h1 class="p1">Ambersoft</h1><br>
+
```
Comment (shivling123) [inline Akash/html/assign 3/Assign9.html]: remove space

File: Akash/html/assign 3/Assign9.html [hunk 36]
```diff
@@ -0,0 +1,39 @@
...
+        <p>
+            
+            <a href="assign8.html">Home | </a>
+            <a href="Assign9.html"> MI-MEAN | </a>
+            <a href="assign10.html"> Contact | </a>
+         </p><br>
+
```
Comment (shivling123) [inline Akash/html/assign 3/Assign9.html]: remove extra space

File: Akash/html/assign 3/Assign9.html [hunk 37]
```diff
@@ -0,0 +1,39 @@
...
+
+        <p1><b>MI-9-MEAN</b></p1>
+        <ul style="list-style-type:circle;">
+            <li>Pawan</li>
+            <li>Hruday</li>
+            <li>Aniket</li>
+        </ul>
+
```
Comment (shivling123) [inline Akash/html/assign 3/Assign9.html]: remove extra space

File: Akash/html/assign 2/Assign2.html [hunk 38]
```diff
@@ -0,0 +1,22 @@
...
+        <h3>
+            Introduction
+        </h3>
+        <p>
+            HTML is a standard markup language for web pages.
+        </p>
+        <h5>Notepad or textedit</h5>
+        <p>Web paged can be created and modified by using professional HTML editors.<br>
+           However,for learning HTML we recommend a simple text editor like Notepad(pc)or textedit(Mac).</p>
+
```
Comment (shivling123) [inline Akash/html/assign 2/Assign2.html]: remove extra space

File: Akash/html/assign 2/assign2.html [hunk 39]
```diff
@@ -0,0 +1,51 @@
...
+    <head>
+        <style>
+            div{background-color: palevioletred;}
+        </style>
+        
+         <title>HTML Assign</title> -->
+    </head>
+    <body>
+        <h1><center>Google</center></h1>
+
```
Comment (shivling123) [inline Akash/html/assign 2/assign2.html]: remove extra space

File: Akash/html/assign 2/assign2.html [hunk 40]
```diff
@@ -0,0 +1,51 @@
...
+        yesr thry struck a.<br>
+        Working about their drom rooms, they built a search engine thats used links to determine the importance of<br>
+    individual pages on the work.<a href="https://www.World Wide web/">World Wide Web</a>
+       <br>
+    They called this search engine backrub.
+     <hr>
+     <h2>Intial public offering</h2>
+     <P>Googel's intial offring (IPO)tooks place five years later,on 19 Augest 2004.<br>
+    At thats larry page,sergy engine birn,and Eric schmidt agreed to work toghster at Google for <strike>30</strike><ims>20</ims><br>
+years, unitl the year 2024.</P>
```
Comment (shivling123) [inline Akash/html/assign 2/assign2.html]: format code

File: Akash/html/assign 2/assign2.html [hunk 41]
```diff
@@ -0,0 +1,51 @@
...
+<pre>Thank You!!</pre>
+<div>
+
```
Comment (shivling123) [inline Akash/html/assign 2/assign2.html]: remove extra space

File: Akash/html/assign 3/assign7.html [hunk 42]
```diff
@@ -5,6 +5,20 @@
             .p1 {
                 font-family: "Times New Roman", 'Times New Roman', Times, serif;
             }
+            ul.a{
+                list-style-type: disc;
+            }
+            ol.b {
+                list-style-type: lower-roman;
+            }
```
Comment (shivling123) [inline Akash/html/assign 3/assign7.html:13]: give space between tag and braces

File: Akash/html/assign 3/assign8.html [hunk 43]
```diff
@@ -3,11 +3,16 @@
...
+    <body>
+        <h2 style="text-align: center;">Ambersoft</h2>
+        <br>
+        <p id="link" style="text-align: center;"> 
     </head>
     <body>
         <h2 style="text-align: center;">Ambersoft</h2>
         <br>
-        <p id="link" style="text-align: center;">
+        <p id="link" style="text-align: center;"
```
Comment (shivling123) [inline Akash/html/assign 3/assign8.html:15]: close p tage 
<p >

File: Akash/html/assign 6/Assignment1.html [hunk 44]
```diff
@@ -0,0 +1,59 @@
...
+            <label type="number">26-40</label>
+        </form>
+        <div>
+            <h2>Academic Detail</h2>
+        </div>
+        <form>
+            <label type="text">Skill set :</label>
+            <input type="checkbox" />
+            <lable type="text">HTML</lable>
+            <input type="checkbox" />
```
Comment (shivling123) [inline Akash/html/assign 6/Assignment1.html:50]: give id to input field and for  to label 
both id and vattribute given tu label must be same
make this change in all code

**Review by shivling123 — CHANGES_REQUESTED** [review]
do changes as soon as and upload it again

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**